packages/local-environment/src/network.ts

```diff
@@ -5,6 +5,7 @@ import { Key } from ".";
 import { sleep } from "./utils";
 import { DockerizedActor } from "./actors";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+// TODO: the old deploy (what is currently published to npm) has been deprecated
```

Comment:
@kekonen can we merge this without these TODOs done, or do they need to be addressed?

Reply:
local-environment seems to be out of date anyways. Would love to delete it in a separate PR

Reply:
PR for deleting local-environment (https://github.com/nomad-xyz/monorepo/pull/266)